src/textual_dev/previews/example_widgets/__init__.py

```diff
@@ -0,0 +1,158 @@
+import random
+from statistics import mean
+
+from textual.app import ComposeResult
+from textual.containers import Container, Center, Middle
+from textual.widgets import (
+    DirectoryTree,
+    Footer,
+    Header,
+    Input,
+    Label,
+    ListView,
+    ListItem,
+    LoadingIndicator,
+    Sparkline,
+    Static,
+    Tree,
+    ProgressBar,
+    TabbedContent,
+    TabPane,
+    Markdown,
+    Tabs,
+    TextLog,
```

Comment:
Remove unused import.
```suggestion
```

Reply:
After I added remaining widgets, [Windows builds started failing](https://github.com/pavelkraleu/textual-dev/actions/runs/5658759006). Probably some added widget is not compatible with Windows?

Removing some widgets was an attempt to investigate which one started causing those problems.

Sorry, this Pull request is still WIP 🙈

Reply:
For me the biggest blockers are those failing [Windows tests](https://github.com/pavelkraleu/textual-dev/actions/runs/5658759006/job/15330783845). Do you know what could be behind that error?  
Otherwise I will disable all newly added widgets one by one and see when tests start passing again. 

```
    def test_cli_widgets():
        runner = CliRunner()
        result = runner.invoke(run, ["widgets"])
>       assert result.exit_code == 0
E       AssertionError: assert 1 == 0
E        +  where 1 = <Result AssertionError('Driver must be in application mode')>.exit_code
```

Reply:
I have tried disabling new widgets. I have no idea why Windows builds fail, I think it is unrelated to my changes.

Reply:
All widgets work on all platforms. Maybe the test is flakey?

Reply:
It seems to be constantly failing.  
Do you know what could `Driver must be in application mode` mean?